Resources/Prototypes/DeltaV/Entities/Objects/Devices/pda.yml

```diff
@@ -199,3 +199,257 @@
   - type: Icon
     sprite: DeltaV/Objects/Devices/pda.rsi
     state: pda-mailcarrier
+
+## Alternate Job Titles
+
+# Passenger
+
+- type: entity
+  parent: PassengerPDA
+  id: VisitorPDA
+  name: visitor PDA
+  description: If you hold onto this, it can be a fun souvenier when you get to... wherever you're going!
+  components:
+  - type: Pda
+    id: VisitorIDCard
+    state: pda
```

Comment:
no need to duplicate state for all these since its inherited

Reply:
Seen, will change and test.

Reply:
Sure enough, you're right. It's been changed.